buildprocess/configureWebpack.js

```diff
@@ -160,6 +162,7 @@ function configureWebpack(
   config.module.rules.push({
     test: /\.(ts|js)x?$/,
     include: [
+      path.resolve(terriaJSBasePath, "node_modules", "commander"),
```

Comment:
These changes are needed to fix a webpack error - there is probably a better way to handle the `commander` path here

<img width="628" alt="image" src="https://github.com/TerriaJS/terriajs/assets/6187649/a3b557f3-a61c-41df-aff6-565d2971467e">